ietf/doc/test_return_to_url.py

```diff
@@ -0,0 +1,37 @@
+# Copyright The IETF Trust 2013-2024, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+import debug                            # pyflakes:ignore
+
+from ietf.utils.test_utils import TestCase
+from .return_to_url import parse_ballot_edit_return_point
+
+
+class ReturnToUrlTests(TestCase):
```

Comment:
Now assuming `return_to_url.py` is empty, make this a class in `tests_ballot.py`?